docs/en/setup/service-agent/java-agent/Supported-list.md

```diff
@@ -42,7 +42,7 @@ metrics based on the tracing data.
   * Mysql Driver 5.x, 6.x, 8.x
   * Oracle Driver (Optional¹)
   * H2 Driver 1.3.x -> 1.4.x
-  * [ShardingSphere](https://github.com/apache/shardingsphere) 3.0.0, 4.0.0, 4.0.1, 4.1.0, 4.1.1, 5.0.0
+  * [ShardingSphere](https://github.com/apache/shardingsphere) 3.0.0, 4.0.0, 4.0.1, 4.1.0, 4.1.1, 5.0.0, 5.1.0, 5.1.1, 5.1.2, 5.2.0, 5.2.1, 5.3.0, 5.3.1, 5.3.2
```

Comment:
Are 5.0.x to 5.2.2 tested?

Reply:
Ok, the other version is indeed not tested, I will remove the other version.

Reply:
We can't accept a lot of codes just for a single version. That is hard to maintain.
ShardingSphere keeps rapid changes and breaking everything.
It is very hard for us to follow. The codebases of ShardingSphere are much bigger than any other frameworks we met.
So, please try to provide a more adoptive version for plugin.

Reply:
You are right that one plugin per version does make maintenance difficult. At present, the old plug-in cannot meet the new version, so a new plug-in needs to be added. In the future, multiple versions of a plugin will be considered.